hr_employee_document/security/security.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="access_hr_employee_system_user_override" model="ir.model.access">
+        <field name="name">hr.employee system user override</field>
+        <field name="model_id" ref="hr.model_hr_employee" />
+        <field name="group_id" ref="base.group_user" />
+        <field name="perm_read" eval="1" />
+        <field name="perm_create" eval="0" />
+        <field name="perm_write" eval="0" />
+        <field name="perm_unlink" eval="0" />
+    </record>
+    <record id="rule_hr_employee_user" model="ir.rule">
+        <field name="name">User can only read their own employees</field>
+        <field name="model_id" ref="hr.model_hr_employee" />
+        <field name="groups" eval="[(4, ref('base.group_user'))]" />
+        <field name="global" eval="True" />
```

Comment:
```suggestion
```

Reply:
`global` is computed, and it will compute to `False` because you have `groups` set.